content/es/talk/example/index.md

```diff
@@ -11,8 +11,8 @@ address:
   street: 450 Serra Mall
 all_day: false
 authors: []
-date: "2030-06-01T13:00:00Z"
```

Comment:
is the example talk needed? In any case it turned up in the RSS feed and RSS feed validators don't seem to like dates in the future :grin:

Reply:
No, it's there because we didn't delete it. I'll accept this change and then remove it.